### DISCUSSION TITLE
Compilation error for UNO start up project

### DISCUSSION THREAD

**Discussion by ApoorvaPetkar** [unoplatform/uno#9367] [Q&A]
Followed this documentation for creating the start-up project - https://platform.uno/docs/articles/get-started-vs-2022.html?tabs=windows

Getting this compile time error-

![image](https://user-images.githubusercontent.com/34087324/180939510-b873f545-4866-41f0-a2a7-35d04e09beae.png)

any idea why?

**Comment by jeromelaban** [comment]
Thanks for the report. Could you tell which version of visual studio are you using ?
